modules/css/icon/stories.tsx

```diff
@@ -16,7 +16,7 @@ const containerStyle = css({
   },
 });
 
-class IconDemo extends Component {
+class IconDemo extends Component<React.PropsWithChildren<any>> {
```

Comment:
```suggestion
class IconDemo extends Component<React.PropsWithChildren> {
```

The default generic for `React.PropsWithChildren` are `unknown`. `unknown` ANDed with any other type is the type.

For example:
```tsx
type Foo = unknown & { children: any } // {children: any}
```

https://www.typescriptlang.org/play?#code/LAKALgngDgpgBAMQPZLgXjgVwHYGttIDu2cAZHAN4DGAFgJYA2AJgE4zYBccAhthAL5wA9EMq1Grdl14CgA